spec/requests/api/v1/documents_controller_destroy_spec.rb

```diff
@@ -53,11 +63,5 @@
 
       it_behaves_like 'an endpoint that responds with error 404'
     end
-
-    context 'with an invalid CONTENT_TYPE header' do
-      let(:content_type) { 'application/xml' }
-
-      it_behaves_like 'an endpoint that responds with error 415'
-    end
```

Comment:
Hi @ngw wouldn't this test still be valid? I think it'll still pass with the changes you've proposed.